paywall/src/__tests__/data-iframe/Mailbox/getBlockchainDataFromLocalStorageCache.test.ts

```diff
@@ -69,6 +69,7 @@ describe('Mailbox - getBlockchainDataFromLocalStorageCache', () => {
         pending: '',
         expired: '',
         confirmed: '',
+        noWallet: '',
```

Comment:
More paywall non-sense. I had to change all of the (unrelated) tests to make sure things worked :/

Reply:
Classic